### DISCUSSION TITLE
征集新图标

### DISCUSSION THREAD

**Discussion by mcthesw** [mcthesw/game-save-manager#34] [Announcements]
新版存档管理器使用了Rust作为开发语言，使用了Tauri框架，那么大家能否为它设计一个图标呢？即使不会写代码，你也可以为该项目作出很大的贡献！

**Comment by kcfuler** [comment]
可以考虑用ai生成一下

**Reply by kcfuler** [reply]
我生成了两张，可以看看

![DALL·E 2023-12-18 04 23 36 - Create a simplified logo for a game-save management software called 'Game-Save-Manager'  The software, developed in Rust and using the Tauri framework](https://github.com/mcthesw/game-save-manager/assets/87866012/4144bb4a-aa6f-403a-83de-dad5687c8b05)
![DALL·E 2023-12-18 04 23 37 - Create a simplified logo for a game-save management software called 'Game-Save-Manager'  The software, developed in Rust and using the Tauri framework](https://github.com/mcthesw/game-save-manager/assets/87866012/467313d9-e1b7-4f3c-85a8-bee55ad03a06)

**Reply by mcthesw** [reply]
看起来还不错，稍加修改就可以使用。不过有一个问题，就是两个图标都包含了云的标识，可能会误导用户以为当前具有云存档功能，你怎么看呢？

**Reply by kcfuler** [reply]
不好意思，前段时间期末，没注意github的消息

我认为这个云的图标主要就是代表存档的保存功能，可以考虑将云替换为类似文件夹的图标

**Reply by mcthesw** [reply]
我很喜欢图2，到时候看看能不能根据它改一张出来

**Reply by mcthesw** [reply]
好消息是现在支持云存档了，可以放心的使用云了
